src/utils/currency.ts

```diff
@@ -143,22 +170,34 @@ export function getCurrencyIdentifier(currency: CurrencyExt): CurrencyIdentifier
     if (isLendToken(currency)) {
         return { lendToken: currency.lendToken.id };
     }
+    if (isStableLpToken(currency)) {
+        return { stableLpToken: currency.stableLpToken.poolId };
+    }
+    if (isStandardLpToken(currency)) {
+        const token0 = getCurrencyIdentifier(currency.lpToken.token0) as StandardPooledTokenIdentifier;
+        const token1 = getCurrencyIdentifier(currency.lpToken.token1) as StandardPooledTokenIdentifier;
+        return { lpToken: [token0, token1] };
+    }
+
     return { token: currency.ticker };
 }
 
 export async function currencyIdToMonetaryCurrency(
-    assetRegistryApi: AssetRegistryAPI,
-    loansApi: LoansAPI,
```

Comment:
This is a nice change, should be easier to close this issue now: https://github.com/interlay/interbtc-api/issues/599